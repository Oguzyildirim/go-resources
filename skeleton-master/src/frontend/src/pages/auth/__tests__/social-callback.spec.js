import { mount } from '@vue/test-utils'
import Vuex from 'vuex'
import VueRouter from 'vue-router'
import Page from '../social-callback'

describe('social-callback.vue', () => {
  let actions
  let store

  beforeEach(() => {
    actions = {
      'page/title': jest.fn(),
      'loading/start': jest.fn(),
      'loading/finish': jest.fn(),
      'auth/socialCallback': jest.fn()
    }
    store = new Vuex.Store({
      actions,
      state: {
        loading: { status: false }
      }
    })
  })

  it('should match snapshot', () => {
    const router = new VueRouter({})
    const w = mount(Page, {
      store,
      router,
      stubs: ['router-link', 'router-view']
    })

    expect(w.html()).toMatchSnapshot()
    w.destroy()
  })
})
