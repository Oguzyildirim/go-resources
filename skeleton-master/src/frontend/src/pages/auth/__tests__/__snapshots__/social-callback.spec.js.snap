// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`social-callback.vue should match snapshot 1`] = `
<section class="container container--fluid">
  <div class="row justify-center">
    <div class="v-card--material pa-3 pa-6 v-card v-sheet theme--light v-card--material--has-heading" style="min-width: 50%;" min-width="50%">
      <div class="d-flex grow flex-wrap">
        <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 info pa-7" style="width: 100%;">
          <div class="display-2 font-weight-light">
            Successfully logged in via <span class="text-capitalize"></span></div>
          <!---->
          <!---->
        </div>
        <!---->
      </div> <span> <!----> <!----></span>
      <div role="alert" class="v-alert v-sheet theme--dark info">
        <div class="v-alert__wrapper"><i aria-hidden="true" class="v-icon notranslate v-alert__icon material-icons theme--dark">$info</i>
          <div class="v-alert__content">
            Please wait while logging you into the system...
          </div>
        </div>
      </div>
      <!---->
      <!---->
    </div>
  </div>
</section>
`;
